applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -339,6 +340,7 @@ def AssembleTestSuites():
     # Constitutive Law tests
     # nightSuite.addTest(TIsotropicDamageSimoJuPSTest('test_execution')) # FIXME: Needs get up to date
     nightSuite.addTest(TSmallDeformationPlasticityTest('test_execution'))
+    nightSuite.addTest(TSimpleJ2PlasticityTest('test_execution')) 
```

Comment:
why to the night suite and not to the small one?

these tests should take nothing right? if so, please add to the small instead of to the nightly